homework/02LessonTwo.js

```diff
@@ -11,237 +11,280 @@ var assert = require('assert');
-        assert.equal(whichSpecies('nemo'), 'fish');
-      });
-      it('should return false if character is anything else', function () {
-        assert.equal(whichSpecies('stitch'), false);
-      });
-    });
-
-    describe('Problem 8: testNumber(number)', function () {
-      it('should be defined and of type function', function () {
-        assert(!(typeof testNumber === 'undefined'));
-        assert(typeof testNumber === 'function');
-      });
-
-      it('should return "divisible by 4" when number is divisible by 4', function () {
-        assert.equal(testNumber(4), 'divisible by 4');
-      });
-      it('should return "divisible by 2" when number is divisible by 2', function () {
-        assert.equal(testNumber(2), 'divisible by 2');
-      });
-      it('should return "divisible by 3" when number is divisible by 3', function () {
-        assert.equal(testNumber(3), 'divisible by 3');
-      });
-      it('should return "divisible by 5" when number is divisible by 5', function () {
-        assert.equal(testNumber(5), 'divisible by 5');
-      });
-    });
-  });
-
-});
+function addThree() {
+    function add(number1, number2, number3) {
+        return number1 + number2 + number3
+
+    }
+    // ****
+    // Concept Checkpoint
+    //
+    // Write your answer in comments
+    //
+    // What is a function? How do you define a function in Javascript?
+    //
+    // Your Answer Goes Here:
+    // A function is a block of code designed to run a certain task. A function is defined by function , a name and //parentheses
+    //
+    // What is a return value?
+    //
+    // Your Answer Goes Here:
+    // The return value stops the execution of the function and returns the value from that function.
+    //
+    // How do you define a named function?
+    //
+    // Your Answer Goes Here:
+    // Function then the name then the execution parentheses then use your open bracket to add your function
+    //
+    // What is a parameter? What is an argument? Is there a difference between the two?
+    //
+    // Your Answer Goes Here:
+    //
+    // Parameters are basically place holders for your arguments while arguments are the actual values.
+
+
+    // ****
+    // Modulo % Operator
+    // ****
+
+    // Problem 5:
+    // isEven(number) is a function that
+    // returns true if number is even (divisible by 2),
+    // else returns false
+    // complete isEven() by returning a boolean expression
+
+    function isEven(number) {
+        if (number % % 2) {
+            return true;
+        } else {
+            return false;
+        }
+
+        // Problem 6:
+        // isDivisibleByThree(number) is a function that
+        // returns true if number is divisible by 3,
+        // else returns false
+        // complete isDivisibleByThree() by returning a boolean expression
+
+        function isDivisibleByThree(number) {
+            if (number % % 3) {
+                return true;
```

Comment:
you only need to use the modulo operator once: `number % 3`

also note that this conditional will evaluate to true no matter what. As long as the `number` entering your function is a Number value, it will always evaluate true - number division evaluates to true.

Open your console and try `!!(5 % 3)` to see what I mean. 

Remember, in this case we're trying to find even or odd numbers by testing whether or not a number divided by 2 / 3 leaves a remainder.